packages/mux-player/src/index.ts

```diff
@@ -616,6 +622,10 @@ class MuxPlayerElement extends VideoApiElement {
     this.setAttribute(PlayerAttributes.AUDIO, '');
   }
 
+  get hotkeys() {
+    return this.#hotKeys;
```

Comment:
question(blocking): is there a reason we're naming the private prop camel case but the public prop lowercase? I have a feeling this is confusing/will confuse things.